app/views/components/_feedback.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="feedback-box">
+  <h3 class="feedback-box__title">
+    <%= title %>
+  </h3>
+  <svg xmlns="http://www.w3.org/2000/svg" class="feedback-box__icon svg-icon svg-icon--speech" focusable="false">
+    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--speech"></use>
+  </svg>
+  <a href="mailto:<%= "#{email}" %>" class="feedback-box__text">
```

Comment:
the quotes in here don't cause a problem? `"mailto:<%= "#{email}" %>"`

Reply:
No, the quotes don't cause a problem.